app/build.gradle.kts

```diff
@@ -34,8 +34,18 @@ android {
         execution = "ANDROIDX_TEST_ORCHESTRATOR"
     }
 
+    signingConfigs {
+        getByName("debug") {
+            storeFile = file("${System.getProperties()["user.home"]}/.android/debug.keystore")
```

Comment:
I'm new to Kotlin DSL in gradle build files, so I may be wrong here but; shouldn't  these wrapping citation marks around `user.home` be escaped? Like so: `file("...[\"user.home\"]...")`, or maybe single citation marks? Or do we concatenate strings like this in Kotlin script?

Reply:
This is accessing the key of a hashtable and in java the equivalent would be getProperties().get("user.home") therefore there is nothing to escape here.

In Kotlin single quotes are used for a single character 'a' and strings use double quotes "hey"

Reply:
Yes, I understand that, but in Java the string above would give you compile errors. I'm surprised it doesn't in this case. Did you intend it to be:

`"${System.getProperties()[" + user.home + "]}/.android/debug.keystore"` or
`"${System.getProperties()[\"user.home\"]}/.android/debug.keystore"`

My concern is that the `"`-character immediately before `user.home` will terminate the string literal that was started with `"${Syst...` immediately after `file(`.

Reply:
It doesn't give me compilation errors and I have to use the debug keystore with the betanet flavour I am running so it's fine but will test out and look at it more deeply and get back to you

Reply:
My script-fu is not as strong yet obviously. If it works then I can't argue against it :) No need for further investigation as far as I'm concerned.